src/main/java/net/sf/jabref/importer/fileformat/MsBibImporter.java

```diff
@@ -51,9 +52,7 @@ public boolean isRecognizedFormat(InputStream in) throws IOException {
          */
         Document docin;
         try {
-            DocumentBuilder dbuild = DocumentBuilderFactory.
-                    newInstance().
-                    newDocumentBuilder();
+            DocumentBuilder dbuild = DocumentBuilderFactory.newInstance().newDocumentBuilder();
```

Comment:
You should not reformat code if you do not change anything else in the class. That leads to `git blame` outputting something wrong.